.github/workflows/smoke-test.yaml

```
@@ -0,0 +1,54 @@
+    runs-on: self-hosted-hoprnet-bigger
+    timeout-minutes: 10
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@f4a75cfd619ee5ce8d5b864b0d183aff3c69b55a # v2.13.1
+        with:
+          disable-sudo: true
+          egress-policy: audit
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          ref: ${{ inputs.source_branch }}
```

actionlint (1.7.8)

property "source_branch" is not defined in object type {image: string}

(expression)